server/public/styles.css

```diff
@@ -241,7 +241,8 @@ select:focus {
   width: 100%;
   height: 100vh;
   background-size: cover;
-  z-index: -1;
+  z-index: -999;
```

Comment:
This is excessive. Try to fix your code so that you only need small z-index values.